chromadb/test/db/test_system.py

```diff
@@ -273,6 +273,16 @@ def test_create_get_delete_collections(sysdb: SysDB) -> None:
     assert by_collection_result == []
 
 
+def test_get_collection_size(sysdb: SysDB) -> None:
+    if not isinstance(sysdb, GrpcSysDB):
+        pytest.skip("Skipping because this functionality is only supported by GrpcSysDB")
```

Comment:
why not support it for single node?

Reply:
This is strictly to support a distributed use case. Also, they have different schemas so we'd be introducing more change for no gain